voc/python/klass.py

```diff
@@ -51,6 +58,10 @@ def __init__(
         self._constructor = None
 
     @property
+    def full_name(self):
+        return '.'.join(self.namespace.split('.')[1:] + [self.name])
```

Comment:
What's the generated class name here? What happens if I compile the following code:
```
class MyClass1:
    class InnerClass
        ...

class MyClass2:
    class InnerClass
        ...
```

Do the two `InnerClass` instances collide?